### PR TITLE
Try: Format block markup for readibility

### LINKED ISSUE #65: Better formatting for block patterns and templates
- Check this file in theme `patterns/left-aligned-cta-image.php`

The code indentation and spacing lack uniformity, resulting in readability challenges and potential errors for developers and maintainers.

**EDIT:** Let's extend this to all templates and patterns and try to indent them to be more readable being mindful that we are not breaking block markup while doing so

### PR DESCRIPTION
**Pull request by MaggieCabrera**
**Description**

Closes https://github.com/WordPress/twentytwentyfour/issues/65

When we first added TT4 to core in beta 1 we had [feedback](https://github.com/WordPress/wordpress-develop/pull/5283#pullrequestreview-1640410597) about the markup formatting, we postponed making those changes because It required nitpicking every pattern PR and we don't have a way to automate this yet. Also, there have been cases in the past where indenting broke blocks int he editor. This might not be the case anymore, so let's find out if that is the case, and if it is, let's document it.

This PR tries to format block markup for readability. It's still a WIP because I haven't tested that this doesn't break any of the templates or patterns. Any help with that would be appreciated, and this PR is prone to need often rebases, but I think this is the time to get this done if any.

**Testing Instructions**

No markup was altered besides changing spacing so all patterns and templates should look the same

### CODE REVIEW

File: parts/header.html [hunk 1]
```diff
@@ -1,13 +1,22 @@
 <!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"20px","bottom":"20px"}}},"backgroundColor":"base","layout":{"type":"constrained"}} -->
-<div class="wp-block-group alignwide has-base-background-color has-background" style="padding-top:20px;padding-bottom:20px"><!-- wp:group {"align":"wide","layout":{"type":"flex","justifyContent":"space-between","flexWrap":"wrap"}} -->
-	<div class="wp-block-group alignwide"><!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|20"},"layout":{"selfStretch":"fit","flexSize":null}},"layout":{"type":"flex"}} -->
-	<div class="wp-block-group"><!-- wp:site-logo {"width":60 } /-->
+<div class="wp-block-group alignwide has-base-background-color has-background"
+	style="padding-top:20px;padding-bottom:20px">
```
Comment (webmandesign) [inline parts/header.html:3]: Haven't tested this, but breaking actual HTML tag could be an issue here.

Comment (MaggieCabrera) [reply parts/header.html:3]: yeah, I suspect this as well

Comment (webmandesign) [reply parts/header.html:3]: FYI, other than this I personally haven't experienced any issue with beautifying block HTML when building my own block theme.

Comment (MaggieCabrera) [reply parts/header.html:3]: I tested and it didn't break!

Comment (webmandesign) [reply parts/header.html:3]: Great to know this. Gutenberg rules :)

**Comment by MaggieCabrera** [comment]
Can we get this one in before we get more conflicts?

**Review by luminuu — APPROVED** [review]
Fixed the last merge conflict. Feel free to directly mention me in a comment, as I prioritize mentions over all other issues/PRs 🙂

**Pull request merged**